src/main/java/Duke.java

```diff
@@ -1,10 +1,258 @@
+     */
+    public static void printClosingSequence(String endLogo) {
+        printSeparator();
+        System.out.println(MESSAGE_CLOSING);
+        System.out.println(endLogo);
+        printSeparator();
+    }
+
+    /**
+     * Removes the command passed into the method and replaces it with white space
+     *
+     * @params userInput  Original string typed by user and to be modified
+     * @params commandToRemove  Command to be removed from string
+     */
+    public static String removeCommandFromInput(String userInput, String commandToRemove) {
+        String modifiedUserInput = userInput.replace(commandToRemove, REPLACEMENT_BLANK_SPACES);
+        return modifiedUserInput.trim();
+    }
+
+    /**
+     * Returns a string array with the task's description and additional information
+     *
+     * @params userInput Original string typed by user and to be modified
+     * @params commandToRemove  Command to be removed from string
+     * @params identifier  Identifier token to split the string with
+     **/
+    public static String[] determineTaskInformation(String userInput, String commandToRemove, String identifier) {
+        String modifiedString = removeCommandFromInput(userInput, commandToRemove);
+        return modifiedString.split(identifier);
+    }
+
+    /**
+     * Adds user input into the task list
+     *
+     * @params tasks  Array of current stored tasks
+     * @params userInput  Original input by user
+     * @params taskCount  Current count of tasks stored
+     */
+    public static void addTodoTask(Task[] tasks, String userInput, int taskCount) {
+        tasks[taskCount] = new Todo(removeCommandFromInput(userInput, COMMAND_TODO));
+    }
+
+    /**
+     * Adds new deadline task into the task list
+     *
+     * @params tasks  Array of current stored tasks
+     * @params userInput  Original input by user
+     * @params taskCount  Current count of tasks stored
+     */
+    public static void addDeadlineTask(Task[] tasks, String userInput, int taskCount) {
+        String description;
+        String by;
+        String[] informationStrings = determineTaskInformation(userInput, COMMAND_DEADLINE, DEADLINE_IDENTIFIER);
+        description = informationStrings[0].trim();
+        by = informationStrings[1].trim();
+        tasks[taskCount] = new Deadline(description, by);
+    }
+
+    /**
+     * Adds new event task into the task list
+     *
+     * @params tasks  Array of current stored tasks
+     * @params userInput  Original input by user
+     * @params taskCount  Current count of tasks stored
+     * */
+    public static void addEventTask(Task[] tasks, String userInput, int taskCount) {
+        String description;
+        String at;
+        String[] informationStrings = determineTaskInformation(userInput, COMMAND_EVENT, EVENT_IDENTIFIER);
+        description = informationStrings[0].trim();
+        at = informationStrings[1].trim();
+        tasks[taskCount] = new Event(description, at);
```

Comment:
Can the separation of the description and the date be done together in a different method?